src/pages/FlowRun/Details-Tile.vue

```diff
@@ -79,6 +79,9 @@ export default {
         space_in_empty_paren: true,
         preserve_newlines: false
       })
+    },
+    handleRefetch() {
+      this.$emit('refetch')
```

Comment:
I think it would be better if this component didn't listen for an event only to emit bubble it up to it's parent. It's probably more of a refactor than is responsible, but ideally we would probably extract the apollo mutations out of LabelEdit so the parent was responsible for actually executing the query, LabelEdit could be a dumb component like a select-box. It could just use a v-model, and when that model changes the parent can trigger the mutation and emit up to it's parent 1 layer that a refetch is advised. 

Looking at LabelEdit, it appears to have some complexity trying to be the LabelEditor for both flows and tasks, which would also be cleaned up by this refactor. But as I mentioned before, this is probably not a responsible change at the moment.